backend/apps/volontulo/models.py

```diff
@@ -164,14 +164,18 @@ def create_new(self):
 
     def determine_action_status(self):
         """Determine action status by offer dates."""
+
         if (
                 (
-                    self.finished_at and
-                    self.started_at < timezone.now() < self.finished_at
+                    self.finished_at is None and
```

Comment:
what will happen when `self.finished_at` and also `self.started_at` will be set to `None`?

Reply:
Can't here use something like:
```
in_range = (self.started_at or date.min) < my_date < (self.finished_at or date.max)
```